docs/_ug/TryingFirstCommand.md

```diff
@@ -1,8 +1,8 @@
 <!-- markdownlint-disable-file first-line-h1 -->
 
-To let you become more familiar with FoodRem, let's practice executing some commands.
+To let you become more familiar with FoodRem, let's practice executing some [[command:commands]].
 
-To start off, let's try out the `new` command! This command lets you add an [Item](#item) to FoodRem.
+To start off, let's try out the `new` command! This command lets you add an [[item:Item]]to FoodRem.
```

Comment:
Should we link this to the Item entry in the glossary, or the Item section of the UG?

Reply:
Assuming that the Item section comes before this part, I think it will be fine to put the glossary link instead (since we recommended the user to have read the UG sequentially).

Reply:
Hmm, I'm inclined to keep it pointing back to the Item section

Reply:
Ok sure. There's no link to Tag (at least for this section). The next time the link to Tag (and its working fine) is in the section about  Tag Commands